src/inpage-content.js

```diff
@@ -0,0 +1,120 @@
+
+const url = window.location.href;
+// const action = url.split('?')[1].split('=')[1];
+
+const action = parseQuery(url).action;
+const btnLink = parseQuery(url).btnlink;
+
+const loginItem = document.getElementById("fbc-login");
+const emailItem = document.getElementById("fbc-email");
+
+function parseQuery(queryString) {
+    var query = {};
+
+    var pairs = queryString.split('?')[1].split('&');
+    for (var i = 0; i < pairs.length; i++) {
+        var splt = pairs[i].split('=');
+        query[splt[0]] = splt[1]
+    }
+    return query;
+}
+
+
+if (action === "login") {
+    loginItem.classList.remove("is-hidden");
+}
+
+if (action === "email") {
+    emailItem.classList.remove("is-hidden");
+}
+
+// Header String
+const fbcTitle = document.querySelector(".fbc-title");
+fbcTitle.innerHTML = browser.i18n.getMessage("facebookContainer");
+
+// Login Strings
+const fbcPromptSubtitleLogin = document.querySelector(".fbc-subtitle-login");
+fbcPromptSubtitleLogin.innerHTML = browser.i18n.getMessage("inPageUI-tooltip-prompt-p1");
+
+const fbcPromptBodyTextLogin = document.querySelector(".fbc-bodytext-login");
+fbcPromptBodyTextLogin.innerHTML = browser.i18n.getMessage("inPageUI-tooltip-prompt-p2");
+
+const fbcPromptAllow = document.querySelector(".fbc-badge-prompt-btn-allow");
+const fbcPromptCancel = document.querySelector(".fbc-badge-prompt-btn-cancel");
+
+fbcPromptAllow.innerHTML = browser.i18n.getMessage("btn-allow");
+fbcPromptCancel.innerHTML = browser.i18n.getMessage("btn-cancel");
+
+
+fbcPromptAllow.addEventListener("click", (e) => {
+
+    if (!e.isTrusted) {
+    // The click was not user generated so ignore
+    e.preventDefault();
+    return false;
+    } 
+
+    // TODO: Add this condition via postMessage
+    // allowClickSwitch = true; 
+    browser.runtime.sendMessage({
+    message: "add-domain-to-list"
+    });
+
+    // Launch facebook authentication
+    parent.postMessage("allowTriggered", "*");
```

Comment:
Question for @groovecoder: Are there any [vulnerabilities from using postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage#security_concerns) we should be guarded against? 

If so, we may have to migrate this back to using the extension [sendMessage](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/sendMessage).